MatrixSDK/MXSession.h

```diff
@@ -1100,4 +1113,22 @@ typedef void (^MXOnBackgroundSyncFail)(NSError *error);
  */
 - (void)removeAllListeners;
 
+#pragma mark - Publicised groups
+
+/**
+ Force the matrix session to refresh the cached data related to the publicised groups of the users.
+ */
+- (void)markOutdatedPublicisedGroupsByUserData;
+
+/**
+ Get the current publicised groups for a matrix user.
+ If this information is not available or marked as outdated, this method returns the current
+ value and triggers a server request to retrieve the actual publicised groups of this user.
+ Listen to `kMXSessionDidUpdatePublicisedGroupsForUsersNotification` to be notified in case of update.
+ 
```

Comment:
Could you also indicate that there will be no http requests if there is already one in progress for this userId?

Reply:
ok